### ISSUE TITLE
[Proposal] Provide TypeScript config for minimum common API

### ISSUE THREAD

**Issue by ytxmobile98** [wintercg/proposal-common-minimum-api#31]
My company is looking for having a [TypeScript config](https://github.com/tsconfig/bases/) for the common minimum API.

So developers may be able to write something like this in tsconfig.json:

```json
{
  "extends": "@tsconfig/wintercg/tsconfig.json"
}
```

This tsconfig should at least support the common minimum APIs which are listed in https://blog.cloudflare.com/introducing-the-wintercg/ and https://common-min-api.proposal.wintercg.org/.

Install the wintercg tsconfig:

```bash
npm install --save-dev @tsconfig/wintercg
yarn add --dev @tsconfig/wintercg
```

This could be useful for developing service workers that are used in Serverless production.

**Comment by mk-pmb** [comment]
> Would also be cool if this could get auto-generated from the specs doc.

Yes. I think we should cooperate with 3rd-party projects to see what they can already generate from the specs, and how we could optimize the specs to help them.
